ol7/templates/template_OVAL_package_required_version_installed

```diff
@@ -0,0 +1,28 @@
+<def-group>
```

Comment:
Don't add this file. Merge this functionality into template_OVAL_package_installed